index.bs

```diff
@@ -798,17 +802,33 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 		{{Clipboard/read()}} returns a {{Promise}} to [=clipboard items=] object that represents contents of [=system clipboard data=].
 		</p>
 
+		An <dfn>unsanitized</dfn> object is a [=sequence=] of {{DOMString}} corresponding to the [=representation/mime type=].
+
+		<p class="note">
+		Only "text/html" MIME type is supported for [=unsanitized=] object. In the future, other MIME types may be added.
```

Comment:
Should we create a section for supported MIME types (with only "text/html" in it for now) and reference it from here and the algorithm below? That will avoid replicating the same information in two places.

Reply:
Done.